README.md

```diff
@@ -1,2 +1,74 @@
-# capstone-project-1
-HTML/CSS &amp; JavaScript capstone project
+# HTML/CSS & JavaScript capstone project - Conference page 🏅!
+
+Online website for the open BIM global conference. This project followed the design guidelines from 
+
+![CC Global Summit 2015 design by Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015])
+
+In this 1st capstone project at microverse the main objectives were:
+
+Main objetives of this new project-feature is:
+
+* Use semantic HTML tags.
+* Apply best practices in HTML code.
+* Use CSS selectors correctly.
+* Use CSS box model.
+* Use Flexbox to place elements in the page.
+* Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
+* Use GitHub Pages to deploy web pages.
+* Apply JavaScript best practices and language style guides in code.
+* Use JavaScript to manipulate DOM elements.
+* Use JavaScript events.
+* Use objects to store and access data.
+
+
+Check out the website [HERE](https://jr-cast.github.io/capstone-project-1/);
+
+## Built With
+
+- HTML
+- Vanilla CSS 
+- Vanilla JavaScript
+- Bootstrap for containers
+
+## Prerequisites
+
+### Setup
+
+- Set up a new repository and prepare it for development using best practices (e.g. linters).
+- Create a new branch according to the GitHub flow rules.
+- Add a .gitignore file.
+- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
+- Enable GH web Page on portfolio repository to deploy it on the web
+- Add descriptive README file to your project
+
+### Install
+
+For instructions of how to install the required dependencies, please visit:
+
+- https://github.com/microverseinc/linters-config
+
+Not familiar with node & npm, please visit:
+
+- https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
+
+Check the templates on Figma website
+
+- https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1
+
+## Authors
+
+:bust_in_silhouette: Jose Ramon
+
+- GitHub: @jr-cast
+- Twitter: @josercastanos
+- LinkedIn: https://www.linkedin.com/in/jr-cast
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+
+Give a :star: if you like this project!
```

Comment:
- [ ] Please, consider that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
✔️ well resolved